lib/zipkin-tracer/zipkin_b3_single_header_format.rb

```diff
@@ -0,0 +1,38 @@
+module ZipkinTracer
```

Comment:
how about adding the magic comment about frozen strings?

Reply:
Added in f8d6a89